res/TensorFlowLiteRecipes/Inf_FullyConnected_000/test.recipe

```diff
@@ -0,0 +1,45 @@
+operand {
+  name: "ifm"
+  type: FLOAT32
+  shape { dim: 1 dim: 2 dim: 3 }
+  shape_signature { dim: -1 dim: 2 dim: 3}
+}
+operand {
+  name: "weight"
+  type: FLOAT32
+  shape { dim: 4 dim: 3 }
+  filler {
+    tag: "gaussian"
+    arg: "0.0"
+    arg: "1.0"
+  }
+}
+operand {
+  name: "bias"
+  type: FLOAT32
+  shape { dim: 4 }
+  filler {
+    tag: "gaussian"
+    arg: "0.0"
+    arg: "1.0"
+  }
+}
+operand {
+  name: "ofm"
+  type: FLOAT32
+  shape { dim: 1 dim: 4 }
+  shape_signature { dim: -1 dim: 4 }
+}
+operation {
+  type: "FullyConnected"
+  fullyconnected_options {
+    activation: NONE
+    keep_num_dims: false
```

Comment:
It is not strictly necessary, but
It might also be a good to add another recipe that corresponds to the case where `keep_num_dims: true`.

Reply:
I will add the case with `keep_num_dims : true`.